src/vs/workbench/contrib/customEditor/browser/customEditorInput.ts

```diff
@@ -135,7 +135,6 @@ export class CustomEditorInput extends LazilyResolvedWebviewEditorInput {
 		let capabilities = EditorInputCapabilities.None;
 
 		capabilities |= EditorInputCapabilities.CanDropIntoEditor;
```

Comment:
The main thing left is how do we decide if a custom/webview editor should support AUX window or not. IIRC we discussed before that we only want to do this for editors are back'ed up by TextDocument as moving to AUX window is a close-then-open operation, for binary custom editor it might lead to data loss.

Reply:
Yeah I posted this in slack already: I think moving a non-dirty editor is probably OK even though you loose the content, considering that the editor can restore the contents since it was not changed. For moving dirty custom editors that are not backed by a file, we probably should ask to save.